docs/news.rst

```diff
@@ -28,6 +28,8 @@ Psycopg 3.2 (unreleased)
   in non-pipeline mode and not totally reliable (:ticket:`#604`).
   The `Cursor` now only preserves the results set of the last
   `~Cursor.execute()`, consistently with non-pipeline mode.
+- Add `~rows.scalar_row()` factory to generate `rows.RowMaker` returning the scalar value
+  taken from the first and only column (:ticket:`#724`).
```

Comment:
I propose the following wording:

```suggestion
- Add `~rows.scalar_row` to return scalar values from a query (:ticket:`#723`).
```

Also I would move this point to line 22, because in this changelist there are first new features, then bugfixes, and roughly in order from the most generally useful to the least, and this feature is pretty useful to several people, more than for instance the features described at line 22.